.github/workflows/ci.yml

```diff
@@ -116,6 +116,7 @@ jobs:
 
     timeout-minutes: 120
     runs-on: ${{matrix.os}}
+    if: github.repository == 'boostorg/do-not-run'
```

Comment:
Is this required here? BTW: GHA supports detection of skip markers in commits

Reply:
this branch doesn't need to run GHA.   how should it be disabled?
Not sure if you are suggesting to always write "[no CI]" in the commit comments, every time.  Or, which skip markers?

Reply:
Ah I misunderstood the intention. I think `if: false` would have worked too.

Reply:
Switched it to `if: false`